tree/ntuple/v7/test/ntuple_processor_join.cxx

```diff
@@ -0,0 +1,181 @@
+#include "ntuple_test.hxx"
+
+#include <ROOT/RNTupleProcessor.hxx>
+
+TEST(RNTupleJoinProcessor, Basic)
+{
+   FileRaii fileGuard("test_ntuple_join_processor_basic.root");
+   {
+      auto model = RNTupleModel::Create();
+      auto fldX = model->MakeField<float>("x");
+      auto ntuple = RNTupleWriter::Recreate(std::move(model), "ntuple", fileGuard.GetPath());
+
+      for (unsigned i = 0; i < 5; ++i) {
+         *fldX = static_cast<float>(i);
+         ntuple->Fill();
+      }
+   }
+
+   std::vector<RNTupleSourceSpec> ntuples;
+   try {
+      auto proc = RNTupleProcessor::CreateJoin(ntuples);
+      FAIL() << "creating a processor without at least one RNTuple should throw";
```

Comment:
Shouldn't it require at least two RNTuples ?

Reply:
In principle the RNTupleProcessor should work with just one ntuple as well. This is allowed in the chain processor, and I don't see a reason why this wouldn't be allowed for the join processor as well. I think it would be nice to have this symmetry between the two.

Reply:
(I had the same question, but already answered above)